app/res/values/styles.xml

```diff
@@ -137,4 +137,33 @@
         <item name="android:textColor">@color/white</item>
     </style>
 
-</resources>
\ No newline at end of file
+    <style name="ListViewV2">

```

Comment:
Is there a better descriptor than V2 for these? Most of these before didn't actually have V1's, did they? I'm fine with just naming them, although context would also be helpful if they're only used in one specific place currently